### DISCUSSION TITLE
Home icon needed

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#1961] [Request: Icon] [answered]
Hi, 

We need a new icon for the Overview menu item (also the homepage) in the navbar for the new Billing Monkey. We're thinking of the '[Home](https://app.streamlinehq.com/icons/streamline-regular/interface-essential/home/house-1)' (thanks for the suggestion @katieporter) icon but are open to recommendations. 

How can we go about getting this added?

![image](https://user-images.githubusercontent.com/72031986/138306352-c01daaf5-6607-4f79-948d-a8cf7112d369.png)

**Comment by loreina** [comment]
Hi @PoojMenon!

I can mock up some options for the icon and follow up for feedback. :)

In general, we have a self-service guide to add icons to Paste if you'd like to contribute in the future: https://paste.twilio.design/icons/how-to-add-an-icon

**Comment by loreina** [comment, marked as answer]
Hi @PoojMenon, the Home icon is now available in Figma as `logo/product/Home`. It was also just merged in #1973 as `ProductHomeIcon`.

**Reply by PoojMenon** [reply]
Thanks very much for the quick turnaround! Really appreciate that!
